src/instructlab/sdg/utils/chunkers.py

```diff
@@ -211,8 +215,11 @@ def chunk_documents(self) -> List:
         if self.document_paths == []:
             return []
 
-        model_artifacts_path = StandardPdfPipeline.download_models_hf()
-        pipeline_options = PdfPipelineOptions(artifacts_path=model_artifacts_path)
+        if not self.docling_model_path.exists():
+            raise FileNotFoundError(f"Docling model path not found: {self.docling_model_path}")
+        print("docling_model_path", docling_model_path)
```

Comment:
here instead of erroring out we can check if `docling_model_path` is `None` and if it is, then perform the model download here

Reply:
see https://github.com/instructlab/sdg/pull/362/files#r1837291550

Reply:
addressed